Prepare provider readmes for 2020.11.07 version

potiuk opened the pull request:
<!--
Thank you for contributing! Please make sure that your code changes
are covered with tests. And in case of new features or big changes
remember to adjust the documentation.

Feel free to ping committers for the review!

In case of existing issue, reference it using one of the following:

closes: #ISSUE
related: #ISSUE

How to write a good git commit message:
http://chris.beams.io/posts/git-commit/
-->

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

potiuk committed:
Prepare provider readmes for 2020.11.07 version

kaxil commented on airflow/providers/elasticsearch/BACKPORT_PROVIDER_CHANGES_2020.11.07.md:
```diff
@@ -0,0 +1,9 @@
+
+
+### Release 2020.11.7
```
Can you remind me again please on why we strip `0` in the date -- I can vaguely remember it had soemthing to do with PyPI release convention / parsing or stripping of 0??

potiuk replied:
Yep. That's it. PyPI won't accept 2020.11.07 as version.

potiuk replied:
And we are releasing 11.13 eventually so no issue whatsoever :)

potiuk committed:
Moves provider packages scripts to dev

The change #10806 made airflow works with implicit packages
when "airflow" got imported. This is a good change, however
it has some unforeseen consequences. The 'provider_packages'
script copy all the providers code for backports in order
to refactor them to the empty "airflow" directory in
provider_packages folder. The #10806 change turned that
empty folder in 'airflow' package because it was in the
same directory as the provider_packages scripts.

Moving the scripts to dev solves this problem.
